data/students.js

```diff
@@ -1,447 +1,471 @@
-    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/269/medium/Emily_Cheng.jpg?1463174090",
+    "id": 1412,
+    "name": "Elias Lee",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/412/medium/Lee.jpg?1468538025",
     "occup": "student"
   },
   {
-    "id": 1231,
-    "name": "Guruprasad Ramprakash",
-    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/231/medium/Guruprasad_Ramprakash.jpg?1463174103",
+    "id": 1390,
+    "name": "Emily Monte Calvo",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/390/medium/Monte.jpg?1468538023",
     "occup": "student"
   },
   {
-    "id": 1261,
-    "name": "Hiroyuki Kajino",
-    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/261/medium/Hiroyuki_Kajino.jpg?1463174113",
+    "id": 1378,
+    "name": "Eric Salazar",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/378/medium/Salazar.jpg?1468538042",
     "occup": "student"
   },
   {
-    "id": 1281,
-    "name": "Jasper Chen",
-    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/281/medium/Jasper_Chen.jpg?1463174129",
+    "id": 1399,
+    "name": "Forest Turner",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/399/medium/Turner.jpg?1468538051",
     "occup": "student"
   },
   {
-    "id": 1252,
-    "name": "Jie Wei",
-    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/252/medium/Jie_Wei.jpg?1463174140",
+    "id": 1419,
+    "name": "Grace Patuwo",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/419/medium/Patuwo.jpg?1468538485",
     "occup": "student"
   },
   {
-    "id": 1292,
-    "name": "Jin Gwoun",
-    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/292/medium/Jin_Gwoun.jpg?1463174199",
+    "id": 1401,
+    "name": "Greg Chamberlain",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/401/medium/Chamberlain.jpg?1468538496",
     "occup": "student"
   },
   {
-    "id": 1298,
-    "name": "Joel Mangin",
-    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/298/medium/Joel_Mangin.jpg?1463174221",
+    "id": 1366,
+    "name": "Halil Coban",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/366/medium/Coban.jpg?1468538507",
+    "occup": "student"
+  },
+  {
+    "id": 1351,
+    "name": "Ian Hall",
+    "imageUrl": "http://s3-us-west-2.amazonaws.com/aa-progress-tracker/students/avatars/000/001/351/medium/Hall.jpg?1468538518",
+    "occup": "student"
+  },
+  {
+    "id": 1349,
+    "name": "ICHUNG LIN",
```

Comment:
Make sure all names are correctly capitalized. This student should probably be changed in Progress Tracker.